docs/tasks/configure-pod-container/configure-liveness-readiness-probes.md

```diff
@@ -0,0 +1,212 @@
+---
+redirect_from:
+- "/docs/user-guide/liveness/"
+- "/docs/user-guide.liveness.html"
+title: Configuring Liveness and Readiness Probes
+---
+
+{% capture overview %}
+
+This page shows how to configure liveness and readiness probes for Containers.
+The [kubelet](/docs/admin/kubelet/)
+performs liveness probes to check the health of a running application.
+The kubelet performs readiness probes to determine whether an application is
+ready to receive traffic.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Defining a liveness command
+
+Many applications running for long periods of time eventually transition to
+broken states, and cannot recover except by being restarted. Kubernetes provides
+liveness probes to detect and remedy such situations.
+
+In this exercise, you create a Pod that runs a Container based on the
+`gcr.io/google_containers/busybox` image. Here is the configuration file for the Pod:
+
+{% include code.html language="yaml" file="exec-liveness.yaml" ghlink="/docs/tasks/configure-pod-container/exec-liveness.yaml" %}
+
+In the configuration file, you can see that the Pod has a single Container.
+The `livenessProbe` field specifies that the kubelet should perform a liveness
+probe every 5 seconds. The `initialDelaySeconds` field tells the kubelet that it
+should wait 10 second before performing the first probe. To perform a probe, the
+kubelet executes the command `cat /tmp/healthy` in the Container. If the
+command returns a success code, the kubelet considers the Container to be alive and
+healthy. If the command returns a failure code, the kubelet kills the Container
+and restarts it.
+
+When the Container starts, it executes this command:
+
+    /bin/sh -c touch /tmp/healthy; sleep 30; rm -rf /tmp/healthy; sleep 600
+
+For the first 30 second of the Container's life, there is `/tmp/healthy` file.
+So during the first 30 seconds, the command `cat /tmp/healthy` returns a success
+code. After 30 seconds, `cat /tmp/healthy` returns a failure code.
+
+Create the Pod:
+
+    kubectl create -f http://k8s.io//docs/tasks/configure-pod-container/exec-liveness.yaml
+
+Within 30 seconds, view the Pod events:
+
+    kubectl describe pod liveness-exec
+
+The output indicates that no liveness probes have failed yet:
+
+    FirstSeen    LastSeen    Count   From            SubobjectPath           Type        Reason      Message
```

Comment:
wrap in triple ticks or are we not doing that. What's nice is it can do syntax highlighting with a clue